### PR TITLE
Feature/routes faq contact

### PR DESCRIPTION
**Pull request by SamDbs**

### CODE REVIEW

File: api/src/models/brand.ts [hunk 1]
```diff
@@ -10,6 +19,15 @@ export default class Brand extends BaseEntity {
   @Column()
   name!: string
 
+  @CreateDateColumn({ name: 'created_at' })
```
Comment (digiz3d) [inline api/src/models/brand.ts:22]: Do we need to rename it to something snakecase ? Can't we let it camelCase like we did with the other fields, for exemple productId ?

**Review by digiz3d — CHANGES_REQUESTED** [review]
Looks gooood 👏 

I think consistency is important. What do you think about my comment below ?

File: api/src/models/brand.ts (hunk 1 above)
Comment (SamDbs) [reply api/src/models/brand.ts]: Yes i made the change, it's now in camelCase as typescript property since we did camelCase for FK fields too.

**Review by digiz3d — APPROVED** [review]
Let's go 🚀

**Pull request merged**